src/v/pandaproxy/schema_registry/configuration.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+#include "config/config_store.h"
+#include "config/configuration.h"
+#include "config/tls_config.h"
+#include "model/metadata.h"
+
+namespace pandaproxy::schema_registry {
+
+/// Schema Registry configuration
+///
+struct configuration final : public config::config_store {
+    config::one_or_many_property<model::broker_endpoint> schema_registry_api;
```

Comment:
do we need some form of `advertised-api` thing here too by chance?

Reply:
> do we need some form of `advertised-api` thing here too by chance?

I'm not sure yet. I haven't really looked into the high availability / clustered modes of operation, but I haven't seen anywhere that would need an advertised address.